terraform/modules/rds/parameters.tf

```diff
@@ -0,0 +1,6 @@
+resource "aws_ssm_parameter" "database_partial_connection_string" {
+  #checkov:skip=CKV_AWS_337:default encryption not using a kms cmk sufficient
+  name  = "DATABASE_PARTIAL_CONNECTION_STRING"

```

Comment:
Renaming this from `DATA_COLLECTOR_DATABASE_URL` makes terraform want to recreate it (which I've generally been trying to avoid), but I think it's ok in this case

Reply:
Yeah, seems fine to let it do that.